Cargo.toml

```diff
@@ -14,9 +14,10 @@ repository = "https://github.com/barafael/mpu6050-dmp-rs"
 readme = "README.md"
 
 [features]
-default = []
+default = ["temperature"]
```

Comment:
I think its safe to assume all the chips support the feature. It doesn't add much code size. And even if some chip doesn't have a thermometer, no harm done. Just add it to the public API. That will also simplify the docs a tiny bit :)